src/vm/node_modules/VM.js

```diff
@@ -11524,6 +12006,13 @@ exports.reprovision = function (uuid, payload, options, callback)
                 cb(err);
             });
         }, function (cb) {
+            // Skip installZone for bhyve (already handled in reprovisionBhyve)
```

Comment:
There MIGHT be a better way to avoid all of these `if !bhyve skip` paths, but that's not a now problem.  LIke the ZFS output, it's something to note.